### PR TITLE
Courier Unicode 1.4 (new formula)

### PR DESCRIPTION
**Pull request by anthonygreen**
A dependency of maildrop http://www.courier-mta.org/maildrop/

### CODE REVIEW

File: Formula/courier-unicode.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    inreplace "courier-unicode.h", "extern const char *unicode_locale_chset_l(locale_t l);", ""
+    system "make"
+    system "make", "install"
+  end
+
+  # https://sourceforge.net/p/courier/mailman/message/34719502/
+  patch :p0, :DATA
+
+  test do
+    system "false"
```
Comment (MikeMcQuaid) [inline Formula/courier-unicode.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/courier-unicode.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                          "--prefix=#{prefix}"
+
+    # https://sourceforge.net/p/courier/mailman/message/34719502/
+    inreplace "courier-unicode.h", "extern const char *unicode_locale_chset_l(locale_t l);", ""
+    system "make"
+    system "make", "install"
+  end
+
+  # https://sourceforge.net/p/courier/mailman/message/34719502/
+  patch :p0, :DATA
```
Comment (MikeMcQuaid) [inline Formula/courier-unicode.rb]: It's not clear to me from these message: was this patch (and the inreplace) merged?

Comment (anthonygreen) [inline Formula/courier-unicode.rb]: @MikeMcQuaid the patches never seem to have been merged. The last commit to SourceForge was in Sept '15 and the thread dates from Dec '15

File: Formula/courier-unicode.rb (hunk 1 above)
Comment (anthonygreen) [inline Formula/courier-unicode.rb]: @svarshavchik can you suggest a suitable test?

Comment (svarshavchik) [inline Formula/courier-unicode.rb]: Anthony Green writes:

> « HTML content follows »
> In <URL:https://github.com/Homebrew/homebrew-core/pull/ 
> 4511#discussion_r77473991>Formula/courier-unicode.rb:
> > +    system "./configure", "--disable-debug",
> > +                          "--disable-dependency-tracking",
> > +                          "--disable-silent-rules",
> > +                          "--prefix=#{prefix}"
> > +
> > +    # https://sourceforge.net/p/courier/mailman/message/34719502/
> > +    inreplace "courier- 
> unicode.h", "extern const char *unicode_locale_chset_l(locale_t l);", ""
> > +    system "make"
> > +    system "make", "install"
> > +  end
> > +
> > +  # https://sourceforge.net/p/courier/mailman/message/34719502/
> > +  patch :p0, :DATA
> > +
> > +  test do
> > +    system "false"
>
> <URL:https://github.com/svarshavchik>@svarshavchik can you suggest a suitable  
> test?

I'm not sure I follow. To test unicode_local_chset_l(), use newlocale() to  
create a locale with a specific character set, which unicode_local_chset_l()  
should return.

File: Formula/courier-unicode.rb (hunk 2 above)
Comment (MikeMcQuaid) [inline Formula/courier-unicode.rb]: I think we'll probably pass on this as it seems to be unmaintained.

Comment (anthonygreen) [inline Formula/courier-unicode.rb]: @MikeMcQuaid the project has moved to Github https://github.com/svarshavchik/courier-libs but I had issues attempting to use that source in a Homebrew formula. Additionally I couldn't get maildrop to compile on OSX and as reason for adding courier-libs was as a upstream dependency I figure we should pass until those issues are resolved

**Comment by anthonygreen** [comment]
Closing pull request until such time as issues with the library and maildrop are resolved

**Pull request closed without merging**